pkgs/development/python-modules/ftfy/default.nix

```diff
@@ -7,18 +7,17 @@
 , python
 , isPy3k
 }:
+
 buildPythonPackage rec {
-  name = "${pname}-${version}";
   pname = "ftfy";
-  # latest is 5.1.1, buy spaCy requires 4.4.3
```

Comment:
Why did you drop the comment?
Please open a PR at the spaCy side that allows it to use a newer ftfy version.

Reply:
I can open a PR but I very much doubt that it will be accepted.
Spacy is still very much supporting python2.7 and ftfy v5 dropped support for python3.
The v4 branch of ftfy still gets some updates so I doubt the maintainer of Spacy will accept working with two different versions of ftfy as that would make keeping the 2.7 compatibility that much harder.

Reply:
As long as nobody needs a newer version of ftfy, I think it's okay to stay at the old one. However, please keep the comment.